share/lib/python/neuron/rxd/rxd.py

```diff
@@ -804,7 +818,7 @@ def _setup_matrices():
                     grid_3d_indices_cnt += cnt_neighbors_3d
                     area = (numpy.pi * 0.25 * hybrid_diams[index1d] ** 2)
                     areaT = sum([v**(2.0/3.0) for v in vols3d])
-                    volumes1d.append(node._volumes[index1d])
+                    volumes1d.append(hybrid_vols1d[index1d])
```

Comment:
Shouldn't hybrid_vols1d and node._volumes be the same at least for the hybrid voxels?

Reply:
No, index1d can be the index to a node with zero volume.